lib/checkPullRequestLabels.js

```diff
@@ -15,6 +35,97 @@ var matchChangelogLabelWithRegex = function(changelogLabel) {
 };
 
 /**
+ * Checks if the project owner can be assigned to this PR.
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ * @param {String} changelogLabel
+ */
+const canAssignProjectOwner = (pullRequest, changelogLabel) => {
+  // Check if the changelog label is excluded from codeowner assignment.
+  const isLabelExcluded = LABELS_EXCLUDED_FROM_CODEOWNER_ASSIGNMENT.some(
+    (label) => changelogLabel.startsWith(label)
+  );
+  if (isLabelExcluded) {
+    return true;
+  }
+  const projectOwner = getProjectOwner(changelogLabel);
+  const reviewers = pullRequest.requested_reviewers.map(
+    reviewer => reviewer.login
+  );
+  const canReview = reviewers.includes(projectOwner);
+
+  return canReview;
+};
+
+/**
+ * Return the project onwer from a changelog label.
```

Comment:
Write docstrings as nouns - This function is used to obtain....

Reply:
Done.